### DISCUSSION TITLE
Option for automatic indentation after line break ?

### DISCUSSION THREAD

**Discussion by niltsz** [fonsp/Pluto.jl#1417] [Questions about using Pluto] [answered]
Rather often I find that my code is too long to fit in one line of the Pluto notebook. Is there an option that, after the line break, the text is indented automatically to match the indentation of the line it belongs to? Or the indentation of the `=` of that current line.

**Comment by BrettKnoss** [comment]
I'm doing something where this would be useful as well.  It would be helpful for people without coding experience.  I would also like to know if it's possible to make a widget that fills out code when details are filled in.

**Comment by BrettKnoss** [comment]
add this to issues, with an "improvement" tag.

**Comment by fonsp** [comment, marked as answer]
Thanks! Can you post this suggestion to our built-in feedback system instead? https://github.com/fonsp/Pluto.jl/issues/182
